bftengine/src/bftengine/ReplicaBase.hpp

```diff
@@ -126,8 +121,7 @@ class ReplicaBase {
 
   ///////////////////////////////////////////////////
   // ControlStateManger
-  std::shared_ptr<ControlStateManager> controlStateManager_;
-  bool stopAtNextCheckpoint_ = false;
+  bool stopAtNextCheckpoint_ = false;  // TODO [TK] remove, use ControlStateManager's state
```

Comment:
I think we already have a cache variable in ControlStateMnager, so we can remove this variable and its uses

Reply:
Removed, just wanted your feedback